markdown/ch1.markdown

```diff
@@ -350,7 +358,10 @@ Pandas is a Python library for data manipulation and analysis,
 with particular emphasis on tabular and time series data.
 Here, we will use it here to read in tabular data of mixed type.
 It uses the DataFrame type, which is a flexible tabular format based on the data frame object in R.
-For example the data we will read has a column of gene names (strings) and multiple columns of counts (integers), so it doesn't make sense to read this data in directly as an ndarray.
+For example the data we will read has a column of gene names (strings) and multiple columns of counts (integers), so reading it into a homogeneous array of numbers would be the wrong approach.
+Although NumPy has some support for mixed data types, it is not designed for
```

Comment:
"just fine" is an overstatement. iiuc, structured datatypes are seen as a single value, right? You lose a lot of the nice things that make NumPy awesome.